policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -50,7 +52,7 @@ struct Namespace {
     service_port_routes: HashMap<ServicePort, ServiceRoutes>,
     /// Stores the route resources (by service name) that do not
     /// explicitly target a port.
-    service_routes: HashMap<String, HashMap<GroupKindNamespaceName, HttpRoute>>,
+    service_routes: HashMap<String, Option<OutboundRouteCollection>>,
```

Comment:
Is a None value ever inserted into this map?

Reply:
Yes, in `Namespace::service_routes_or_default`. `Option<OutboundRouteCollection>` ended up being easier than having an `Empty` variant in `OutboundRouteCollection` cause it lets us use the normal `Option` handling methods.